### PR TITLE
Create login UI

### PR DESCRIPTION
**Pull request by Jorque**
Login UI done. Currently expecting user to enter email and password.

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/drone3d/LoginActivity.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package ch.epfl.sdp.drone3d
+
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import android.widget.EditText
+import android.widget.ProgressBar
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+
```
Comment (GabrielFleischer) [inline app/src/main/java/ch/epfl/sdp/drone3d/LoginActivity.kt:10]: The class should be documented (as everything is private or override, only the top comment is needed)

File: app/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        android:layout_height="wrap_content"
+        android:layout_marginTop="150dp"
+        android:text="@string/email_text"
+        android:textAlignment="viewStart"
+        android:textColor="@color/black"
+        android:textSize="18sp"
+        app:layout_constraintEnd_toEndOf="@id/passwordText"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <EditText
```
Comment (GabrielFleischer) [inline app/src/main/res/layout/activity_login.xml:21]: Cirrus finds some issues in the file

**Review by GabrielFleischer — CHANGES_REQUESTED** [review]
The LoginActivity should be documented and there are some warnings raised by cirrus.
But otherwise, greate quality code.

File: app/src/main/res/layout/activity_login.xml (hunk 2 above)
Comment (Jorque) [reply app/src/main/res/layout/activity_login.xml:22]: About the missing autofillHints attribute, it is set in the login activity file instead. (L.37-38 of LoginActivity.kt)

**Review by GabrielFleischer — APPROVED** [review]

**Review by Anghul — APPROVED** [review]

**Pull request merged**